romfs/lang/de.json

```diff
@@ -148,39 +148,39 @@
+            "credits": "Anerkennung"
         },
         "nca_warn": {
             "title": "Warnung!",
-            "desc": "Einige installierbare Dateien können bösartige Inhalte enthalten! Deaktiviere\ndiese Funktion nur, wenn du absolut sicher bist, dass die zu installierende\nSoftware vertrauenswürdig ist! Willst du die NCA-Signaturprüfung immer noch deaktivieren?",
-            "opt1": "Ja, ich möchte mein Gerät zerstören"
+            "desc": "Einige installierbare Dateien können bösartige Inhalte enthalten! Deaktiviere\ndiese Funktion nur, wenn du absolut sicher bist. Dass die zu installierende\nSoftware vertrauenswürdig ist! Willst du die NCA-Signaturprüfung immer noch deaktivieren?",
+            "opt1": "Ja, ich möchte mein Gerät bricken"
         },
-        "sig_hint": "Gebe eine URL an um meine Signatur Patches zu beziehen",
+        "sig_hint": "Gebe eine URL an, um Signatur Patches zu beziehen",
         "update": {
             "title": "Neue Version verfügbar",
             "desc0": "Awoo Installer ",
-            "desc1": " ist jetzt verfügbar! Bereit zu aktuallisieren?",
-            "opt0": "Aktuallisieren",
-            "top_info": "Aktuallisierung zu Awoo Installer ",
+            "desc1": " ist jetzt verfügbar! Bereit zu aktualisieren?",
+            "opt0": "Aktualisieren",
+            "top_info": "Aktualisierung zu Awoo Installer ",
             "bot_info": "Beziehe Awoo Installer ",
             "bot_info2": "Extrahiere Awoo Installer ",
-            "complete": "Aktuallisierung vollständig!",
-            "failed": "Aktuallisierung fehlgeschlagen!",
-            "end_desc": "Die Software wird nun geschlossen.",
-            "title_check_fail": "Keine Aktuallisierungen gefunden",
+            "complete": "Aktualisierung vollständig!",
+            "failed": "Aktualisierung fehlgeschlagen!",
+            "end_desc": "Die Software wird nun beendet.",
+            "title_check_fail": "Keine Aktualisierungen gefunden",
             "desc_check_fail": "Du bist auf der aktuellsten Version des Awoo Installer!"
         },
         "credits": {
             "title": "Vielen Dank an die folgenden Leute!",
-            "desc": "- HookedBehemoth für NE MENGE Beiträge\n- Adubbz und andere Mitwirkende für Tinfoil\n- XorTroll für Plutonium und Goldleaf\n- blawar (Frauenwürger) und nicoboss für NSZ Unterstützung\n- Die netten Leute auf AtlasNX' Discuck (Immerhin ein paar davon)\n- Die netten Leute auf RetroNX' Discuck (ohne direkte Beteiligung)\n- namako8982 für die Momiji-Kunst\n- TheXzoron dafür das er ein baka ist"
+            "desc": "- HookedBehemoth für EINE MENGE Beiträge\n- Adubbz und andere Mitwirkende für Tinfoil\n- XorTroll für Plutonium und Goldleaf\n- blawar (Frauenwürger) und nicoboss für NSZ Unterstützung\n- Die netten Leute auf AtlasNX' Discuck (Immerhin ein paar davon)\n- Die netten Leute auf RetroNX' Discuck (ohne direkte Beteiligung)\n- namako8982 für die Momiji-Kunst\n- TheXzoron dafür das er ein baka ist"
```

Comment:
I'd rather change `EINE` to `eine` as German doesn't emphasises the same way English does.

Reply:
I followed the english one, there is it also in upper cases written